pmlc/dialect/pxa/tests/resize_tmps.mlir

```diff
@@ -44,3 +44,37 @@ func @inner_indexes(%I: memref<100x100xf32>) -> (memref<100x100xf32>) {
   }
   return %O4 : memref<100x100xf32>
 }
+
+func @no_resize_return() -> memref<10x10xf32> {
+// CHECK-LABEL: func @no_resize_return
+  %cst = constant 0.000000e+00 : f32
+  %0 = alloc() : memref<10x10xf32>
+  // CHECK: %0 = alloc() : memref<10x10xf32>
```

Comment:
Don't use `%0` in lit tests because the name of vars are subject to change and cause the test to fail. Instead, use:

```
// CHECK: %{{.*}} = alloc() : memref<10x10xf32>
```

Or even better, since the test doesn't need to use this var:

```
// CHECK: alloc() : memref<10x10xf32>
```

Reply:
Done.